core/src/main/scala/sbtorgpolicies/libraries.scala

```diff
@@ -23,39 +23,39 @@ object libraries {
   val v47: Map[String, String] = Map[String, String](
     "iota"          -> "0.2.0",
     "case-classy"   -> "0.4.0",
-    "fetch"         -> "0.6.2",
+    "fetch"         -> "0.6.3",
     "freestyle"     -> "0.3.1",
     "github4s"      -> "0.15.0",
     "org-policies"  -> sbtorgpolicies.BuildInfo.version,
     "scheckToolbox" -> "0.2.2"
   )
 
   protected val vOthers: Map[String, String] = Map[String, String](
-    "akka"                     -> "2.5.3",
-    "akka-http"                -> "10.0.9",
+    "akka"                     -> "2.5.4",
+    "akka-http"                -> "10.0.10",
     "algebra"                  -> "0.7.0",
-    "alleycats"                -> "0.1.9",
-    "apache-kafka"             -> "0.11.0.0",
-    "aws-sdk"                  -> "1.11.168",
+    "alleycats"                -> "0.2.0",
+    "apache-kafka"             -> "0.11.0.1",
+    "aws-sdk"                  -> "1.11.197",
     "base64"                   -> "0.2.3",
     "cassandra-driver"         -> "3.3.0",
     "catalysts"                -> "0.0.5",
-    "catbird"                  -> "0.15.0",
+    "catbird"                  -> "0.18.0",
     "cats"                     -> "0.9.0",
-    "cats-effect"              -> "0.3",
+    "cats-effect"              -> "0.4",
     "circe"                    -> "0.8.0",
     "config"                   -> "1.3.1",
     "coursier"                 -> "1.0.0-RC11",
     "discipline"               -> "0.8",
-    "doobie"                   -> "0.4.1",
+    "doobie"                   -> "0.4.4",
     "embedded-redis"           -> "0.6",
     "export-hook"              -> "1.2.0",
     "finch"                    -> "0.15.1",
     "fs2"                      -> "0.9.7",
-    "fs2-cats"                 -> "0.3.0",
+    "fs2-cats"                 -> "0.4.0",
     "h2"                       -> "1.4.196",
-    "http4s"                   -> "0.17.0-M3",
-    "joda-convert"             -> "1.8.2",
+    "http4s"                   -> "0.18.0-M1",
```

Comment:
http4s 0.18.0-M1 depends on fs2 0.10.0-M6 and I'm not sure that it's binary compatible with fs2 0.9.7. It is okay?

Reply:
Good catch, thanks for pointing it out @guersam 

At this moment, this plugin sadly is not checking if libraries are binary compatible. However, when you are using the `sbt-org-policies` plugin and its dependency dsl, you can specify the version which fits better for your project.

For example:

`%%("http4s")` => by default will use the last one: `"0.18.0-M1"`

But you could use it in this way, to force the use of a different version of the artifact:

`%%("http4s", "0.17.0-M3")`

Reply:
I created an issue because it would be great that plugin check it automatically: https://github.com/47deg/sbt-org-policies/issues/638